tests/test_wave_01.py

```diff
@@ -145,8 +146,9 @@ def test_moves_movie_from_watchlist_to_watched():
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][1] == movie_to_watch
```

Comment:
Nice way to check all the dictionary values at once! 

If the order of the `watched` list couldn't be guaranteed we could write this in a way that's more flexible with:
```suggestion
    assert movie_to_watch in updated_data["watched"]
```

Reply:
I love this! Thank you!